### DISCUSSION TITLE
Wildcards in Query Field Names

### DISCUSSION THREAD

**Discussion by st11x** [directus/directus#12984] [Archive]
Hi,

On this page 

https://docs.directus.io/reference/query/#fields

It says you can use wildcards to select fields at each level. Is there a way to specify a wildcard on the field names at a specific level? For example, I have a whole set of meta data fields like meta_title and  meta_description. I would like to return all meta_* fields.

I could put the meta fields into its own collection but I like to keep them in the same collection under its own group. 

Thanks
Matt

**Comment by benhaynes** [comment]
Not as of now, putting them into their own collection relationally would be the only way. It's a good idea though!

**Reply by st11x** [reply]
Thanks for the confirmation.
